runtime/src/main/java/org/corfudb/recovery/FastObjectLoader.java

```diff
@@ -358,15 +364,15 @@ private void handleRetry() {
         retryIteration++;
         if (retryIteration > numberOfAttempt) {
             log.error("processLogData[]: retried {} number of times and failed", retryIteration);
-            throw new RuntimeException("FastSmrMapsLoader failed after too many retry (" + retryIteration + ")");
+            throw new RuntimeException("FastObjectLoader failed after too many retry (" + retryIteration + ")");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)